opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -66,7 +67,8 @@ namespace Opm {
             AllProperties = IntProperties | DoubleProperties
         };
 
-        EclipseState(std::shared_ptr< const Deck > deck , const ParseContext& parseContext);
+        EclipseState(std::shared_ptr< const Deck > );
+        EclipseState(std::shared_ptr< const Deck > , ParseContext );
```

Comment:
why does ParseContext needs to be copied here instead of passing a const reference?